src/App.jsx

```diff
@@ -1,70 +1,81 @@
-import './App.scss';
+import "bulma/css/bulma.css";
+import React, { useState } from "react";
+import "./App.scss";
+import classNames from "classnames";
 
 export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  "Dumplings",
+  "Carrot",
+  "Eggs",
+  "Ice cream",
+  "Apple",
+  "Bread",
+  "Fish",
+  "Honey",
+  "Jam",
+  "Garlic",
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const deleteGood = () => {
+    setSelectedGood("");
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const addGood = (good) => {
+    setSelectedGood(good);
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : "No goods selected"}
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={deleteGood}
+            />
+          )}
+        </h1>
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = good === selectedGood;
 
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                id="selected"
+                key={good}
+                data-cy="Good"
+                className={classNames({
+                  "has-background-success-light": isSelected,
+                })}
+              >
+                <td>
+                  <button
+                    data-cy={isSelected ? "RemoveButton" : "AddButton"}
+                    type="button"
+                    className={isSelected ? "button is-info" : "button"}
```

Comment:
Would be better to use classnames here - https://www.npmjs.com/package/classnames

Reply:
Thnx for your advice, I did it, but i don't understand, why is it better to use classNames? What's the main difference?

Reply:
It's easier for understanding
And when your code grow and you need to add more classes with different conditions, it will be easier to extend it